### PR TITLE
#160792441 Link UI pages

### PR DESCRIPTION
**Pull request by d-rita**
**What the PR does**
Elicit review about linking UI pages.

**Description of Tasks**
- Link user-login page to user-dashboard.
- Link user-login page to admin-login.
- Link admin-login page to admin-dashboard.
- Provide a logout link to connect back to the login pages.

**Relevant Pivotal Tracker Story**
#160792441

**Screenshots**
a
b
c
d

### CODE REVIEW

**Comment by billkabanga** [comment]
Good work linking your pages in your code . 
How can this be tested though?

**Comment by aayebare** [comment]
The UI design looks simple and easy to use. Nice work

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+**Fast-Food-Fast**
+
+Fast-Food-Fast is a food delivery service app for a restaurant.
+
+The UI is currently hosted on Github Pages at https://d-rita.github.io/fast-food-fast/UI
```
Comment (kdaglas) [inline README.md:5]: yea now this is great... for a person interested in your project they can now access the UI without having to first scroll down to find it... just to add on the good readme tricks, use the hashtag for your titles ... so as to create the bold feel to it

**Comment by kdaglas** [comment]
Reall pretty and simple UI

**Comment by nanfuka** [comment]
pages look user friendly. great work

File: README.md (hunk 1 above)
Comment (d-rita) [reply README.md:5]: Okay. Will do. Thank you.

**Review by Sebuliba-Adrian — APPROVED** [review]
LGTM 👍

**Pull request merged**